app/models/calculator/pageviews_per_visit.rb

```diff
@@ -0,0 +1,23 @@
+class Calculator::PageviewsPerVisit
+  attr_reader :pageviews, :unique_pageviews
+
+  def self.calculate(*args)
+    new(*args).calculate
+  end
+
+  def initialize(pageviews:, unique_pageviews:)
+    @pageviews = pageviews
+    @unique_pageviews = unique_pageviews
+  end
+
+  def calculate
+    return nil if pageviews.blank? || unique_pageviews.blank?
+
+    pageviews_float = pageviews.to_f
+    unique_pageviews_float = unique_pageviews.to_f
```

Comment:
Could we move this to the initialiser?

Reply:
I don't think so because it disrupts the `return 0 if ...blank?` checks. `nil.to_f` will always return 0.0 https://ruby-doc.org/core-2.5.0/NilClass.html#method-i-to_f